installer/conf/omsagent.conf

```diff
@@ -89,6 +89,19 @@
   retry_wait 30s
 </match>
 
+<match oms.operation.**>
```

Comment:
This can go in monitor.conf as well.
Since the omsagent.d conf files are included at the top of omsagent.conf, `match oms.operation.**` will still come before the more generic `match oms.**`